lxd/api.go

```diff
@@ -157,9 +166,14 @@ func restServer(d *Daemon) *http.Server {
 		w.Header().Set("Content-Type", "application/json")
 
 		ua := r.Header.Get("User-Agent")
-		if uiEnabled && strings.Contains(ua, "Gecko") {
-			// Web browser handling.
-			http.Redirect(w, r, "/ui/", http.StatusMovedPermanently)
+		if strings.Contains(ua, "Gecko") {
+			if !uiEnabled {
+				w.Header().Set("Content-Type", "text/html")
+				w.WriteHeader(http.StatusOK)
```

Comment:
Actually, how would you feel to just do a redirect here (as in line 175) and then let the `/ui` handler above deal with displaying the message?

That way the `/` endpoint would always return http.StatusMovedPermanently if UA contains "Gecko" and http.StatusOK if not. 

That way monitoring systems can choose not to follow the redirect if they are not concerned about the UI being enabled or not.